src/serverless-functions/send-to-phone.ts

```diff
@@ -0,0 +1,67 @@
+import "source-map-support/register";
+
+import { EFNYC_HOST, serverlessRollbarHandler } from "../utils/serverless-util";
+import { Twilio } from "twilio";
+
+const validatePhoneNumber = (phone: string) => {
+  const phoneDigits = phone.replace(/\D/g, "");
+  if (phoneDigits.length === 11) {
+    return "+" + phoneDigits;
+  } else return false;
+};
+
+const validatePath = (path: string) => (path.slice(0, 1) === "/" ? path : null);
+
+export const handler = serverlessRollbarHandler(async (event) => {
+  const {
+    TWILIO_ACCOUNT_SID,
+    TWILIO_AUTH_TOKEN,
+    TWILIO_PHONE_NUMBER,
+  } = process.env;
+
+  if (!(TWILIO_ACCOUNT_SID && TWILIO_AUTH_TOKEN && TWILIO_PHONE_NUMBER)) {
+    throw new Error("Twilio credentials are not in environment!");
+  }
+
+  const userPhone = event.body && JSON.parse(event.body).userPhone;
```

Comment:
One thing to keep in mind is that the event body might not actually be JSON, so this might throw an exception. Ideally we should return a 400 (bad request) or something similar in such a case, rather than a 500 internal server error, but bleh whatever.

Also keep in mind that JSON can be any kind of object!  It could be just `null`, in which case the `JSON.parse()` will succeed but getting `userPhone` will throw an exception, or `userPhone` could be something other than a string, etc.  If this were an API that lots of different clients were using, we might want to use something like [JSON schema](https://json-schema.org/) to make sure the deserialized object had the right shape but here it's probably not a big deal.

(Actually this is sort of a decent reason for only accepting `application/x-www-form-urlencoded` payloads, since that only supports string keys and values, so it means there's way less validation to do, but ehh don't worry about it here.)